less/navigation.less

```diff
@@ -1 +1,29 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+
+
+.nav-and-h1 {
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 2px dashed lightgrey;
+    margin-left: 25%;
```

Comment:
For more info, check my comment in `home-page.less`. You can remove all of the margins here except for the a tag, and add these properties inside .nav-and-h1:
```
max-width: 800px;
width: 100%;
margin: 0 auto;
```

- To center everything vertically, add in an `align-items: center;`
- To add that extra spacing on the top and bottom, use `padding: 15px 0;`